pages/glossary/whitespace.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Whitespace
+key: whitespace
+---
+
+Whitespace are characters in the [Unicode separator categories](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153).
```

Comment:
Seeing how this will be used in relation to things that aren't accessible names, the definition must also encompass e.g. control characters that are whitespace, such as tabs, newlines, carriage returns etc.

Reply:
We might actually be able to simply rely on the Unicode `White_Space` property: https://www.unicode.org/Public/UCD/latest/ucd/PropList.txt

Reply:
@WilcoFiers, @kasperisager and I discussed and agreed to go ahead with @kasperisager's suggestion + examples of categories/characters.

Reply:
@kasperisager and @WilcoFiers, the whitespace definition has been updated as agreed and is ready for you guys to have a look at it.